app/styles/app.scss

```diff
@@ -1,3 +1,5 @@
+/*importing all required libraries and file */
```

Comment:
until you didn't write that, no one knew that @import, import libraries, required. Add this comment in like 400 more files to it more clear.